### PR TITLE
 Remove mocks for transaction manager and transaction queue

### PR DESCRIPTION
**Pull request by dshulyak**
It is very unlikely that there will be 2 or more implementations of tx manager and queue, as they are tailored specifically to status project requirements.

Blocked by: https://github.com/status-im/status-go/pull/530

### CODE REVIEW

**Review by divan — APPROVED** [review]
100 x 👍 

We definitely need to clean up overused and autogenerated without a valid reason mocks, so I'm super happy to see this PR.

File: geth/transactions/queue/queue.go [hunk 1]
```diff
@@ -8,17 +8,14 @@ import (
...
 	"github.com/ethereum/go-ethereum/accounts/keystore"
 	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/geth/account"
 	"github.com/status-im/status-go/geth/common"
 	"github.com/status-im/status-go/geth/log"
 )
 
 const (
 	// DefaultTxQueueCap defines how many items can be queued.
 	DefaultTxQueueCap = int(35)
```
Comment (themue) [inline geth/transactions/queue/queue.go]: Interesting. Isn't `int` the default here and casting only needed when explicitly wanting something like `uint64` or similar.

File: geth/transactions/queue/queue.go [hunk 2]
```diff
@@ -189,105 +160,77 @@ func (q *TxQueue) Get(id common.QueuedTxID) (*common.QueuedTx, error) {
 	if tx, ok := q.transactions[id]; ok {
 		return tx, nil
 	}
+	return nil, ErrQueuedTxIDNotFound
+}
+
+// LockInprogress returns transcation and locks it as inprogress
+func (q *TxQueue) LockInprogress(id common.QueuedTxID) (*common.QueuedTx, error) {
```
Comment (themue) [inline geth/transactions/queue/queue.go]: `LockInprogress()` or `LockInProgress()`?

**Review by themue — APPROVED** [review]
Yes, wonderful simplification and good step in dropping these autogenerated mocks.

**Comment by themue** [comment]
@dshulyak Will you take care of the conflicts, please? Thx.

**Comment by dshulyak** [comment]
@themue rebased, please consider merging it

**Pull request merged**